pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -350,3 +350,75 @@ func CreateObjects(client *client.Client, objects []runtime.Object) {
 	}
 	fmt.Println("\nApplication has been deployed to Kubernetes. You can run 'kubectl get deployment,svc' for details.")
 }
+
+func DeleteObjects(client *client.Client, name string) {
+	err := client.Services(api.NamespaceDefault).Delete(name)
+	if err != nil {
+		logrus.Fatalf("Error: '%v' while deleting service: %s", err, name)
+	}
+	logrus.Infof("Successfully deleted service: %s", name)
+
+	err = client.Deployments(api.NamespaceDefault).Delete(name, nil)
```

Comment:
Need confirmation @janetkuo / @kadel : We can use:
- ServiceReaper to delete svc
- DeploymentReaper to delete pod, rc, replicaset & deployment
- DaemonReaper to delete daemonset
- JobReaper to delete job. 

Am I missing or wrong something?